### DISCUSSION TITLE
can we test the beta mode ourselves?

### DISCUSSION THREAD

**Discussion by yash-blip** [lysep-corp/ButOSX#15] [Q&A] [answered]
if i build the code using all the data provided will i be able to test it on my own. like to open the menu is there any certain buttons etc?. also while building it via code any thing i need to know about. a small info on this subject will be helfull.

**Comment by Lyceion** [comment, marked as answer]
> if i build the code using all the data provided will i be able to test it on my own. like to open the menu is there any certain buttons etc?. also while building it via code any thing i need to know about. a small info on this subject will be helfull.

Yes, you are able to build the beta mode yourself. But i don't recommand because we are testing some experimental things on beta branch. I definitely recommend build main branch. But it's your choice. 

For open menu, press "insert". I'm gonna change the button but it's easer to use it like that now so...

If you clone repostory with submodules and build it with default settings, it will build without any errors. Everything is ready to compile & go. But you can use the version on releases section is better. Less things to do just download & inject. But as i said before it's your choice.

Edit: if you want to change menu key, just go to the: https://github.com/Lyceion/ButOSX/blob/master/ButOSX/SDK/Hooks/OpenGLHooker/OpenGLHooker.cpp#L288 and change the key id. (Insert is = 73)
